client/src/related_items/Card.jsx

```diff
@@ -4,15 +4,18 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faStar as solidStar } from '@fortawesome/free-solid-svg-icons'
 import { faStar as emptyStar } from '@fortawesome/free-regular-svg-icons'
 
+import ModalWindow from './ModalWindow';
 import helpers from './helpers';
 import './styles/Card.css';
 
 const Card = (props) => {
+  const currentOverviewId = props.currentId;
```

Comment:
What's the difference between these 2 ids

Reply:
One is the current product id in the overview section, the other one is the product id of the card being clicked. I can rename them for more clarity